### ISSUE TITLE
run_tf_ner.py cannot run

### ISSUE THREAD

**Issue by jx669** [huggingface/transformers#4662]
# 🐛 Bug

## Information

Model I am using (Bert, XLNet ...): Bert

Language I am using the model on (English, Chinese ...):bert-base-multilingual-cased

The problem arises when using:
* [ ] the official example scripts: (give details below)
original code in _run_tf_ner.py_
original data from _germaeval2014eval_
original data process from _preprocess.py_

* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
_germaeval2014eval_ NER
* [ ] my own task or dataset: (give details below)

## To reproduce

1. this is the structure
--data
---train.txt
---dev.txt
---test.txt
---label.txt
run.sh (see below)
run_tf_ner.py (orignal)
utils_ner.py (original)
preprocess.py(original)

2. run.sh
export MAX_LENGTH=128
export BERT_MODEL=bert-base-multilingual-cased
export OUTPUT_DIR=germeval-model
export BATCH_SIZE=32
export NUM_EPOCHS=1
export SAVE_STEPS=750
export SEED=1
export data_dir=data

python3 run_tf_ner.py \
--data_dir . \
--labels $data_dir/labels.txt \
--model_name_or_path $BERT_MODEL \
--output_dir $OUTPUT_DIR \
--max_seq_length  $MAX_LENGTH \
--num_train_epochs $NUM_EPOCHS \
--per_gpu_train_batch_size $BATCH_SIZE \
--save_steps $SAVE_STEPS \
--seed $SEED \
--do_train \
--do_eval \
--do_predict

3. sh run.sh will return the invalid configuration error

```
--05/29/2020 11:21:48 - INFO - __main__ -   n_gpu: 2, distributed training: True, 16-bits training: False
05/29/2020 11:21:48 - INFO - __main__ -   Training/evaluation parameters TFTrainingArguments(output_dir='germeval-model', overwrite_output_dir=False, do_train=True, do_eval=True, do_predict=True, evaluate_during_training=False, per_gpu_train_batch_size=32, per_gpu_eval_batch_size=8, gradient_accumulation_steps=1, learning_rate=5e-05, weight_decay=0.0, adam_epsilon=1e-08, max_grad_norm=1.0, num_train_epochs=1.0, max_steps=-1, warmup_steps=0, logging_dir=None, logging_first_step=False, logging_steps=500, save_steps=750, save_total_limit=None, no_cuda=False, seed=1, fp16=False, fp16_opt_level='O1', local_rank=-1, tpu_num_cores=None, tpu_metrics_debug=False, optimizer_name='adam', mode='text-classification', loss_name='SparseCategoricalCrossentropy', tpu_name=None, end_lr=0, eval_steps=1000, debug=False)
05/29/2020 11:21:50 - INFO - transformers.configuration_utils -   loading configuration file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-multilingual-cased-config.json from cache at /home/ll/.cache/torch/transformers/45629519f3117b89d89fd9c740073d8e4c1f0a70f9842476185100a8afe715d1.65df3cef028a0c91a7b059e4c404a975ebe6843c71267b67019c0e9cfa8a88f0
05/29/2020 11:21:50 - INFO - transformers.configuration_utils -   Model config BertConfig {
  "architectures": [
    "BertForMaskedLM"
  ],
  "attention_probs_dropout_prob": 0.1,
  "directionality": "bidi",
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "id2label": {
    "0": "B-LOC",
    "1": "B-LOCderiv",
    "2": "B-LOCpart",
    "3": "B-ORG",
    "4": "B-ORGderiv",
    "5": "B-ORGpart",
    "6": "B-OTH",
    "7": "B-OTHderiv",
    "8": "B-OTHpart",
    "9": "B-PER",
    "10": "B-PERderiv",
    "11": "B-PERpart",
    "12": "I-LOC",
    "13": "I-LOCderiv",
    "14": "I-LOCpart",
    "15": "I-ORG",
    "16": "I-ORGderiv",
    "17": "I-ORGpart",
    "18": "I-OTH",
    "19": "I-OTHderiv",
    "20": "I-OTHpart",
    "21": "I-PER",
    "22": "I-PERderiv",
    "23": "I-PERpart",
    "24": "O"
  },
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "label2id": {
    "B-LOC": 0,
    "B-LOCderiv": 1,
    "B-LOCpart": 2,
    "B-ORG": 3,
    "B-ORGderiv": 4,
    "B-ORGpart": 5,
    "B-OTH": 6,
    "B-OTHderiv": 7,
    "B-OTHpart": 8,
    "B-PER": 9,
    "B-PERderiv": 10,
    "B-PERpart": 11,
    "I-LOC": 12,
    "I-LOCderiv": 13,
    "I-LOCpart": 14,
    "I-ORG": 15,
    "I-ORGderiv": 16,
    "I-ORGpart": 17,
    "I-OTH": 18,
    "I-OTHderiv": 19,
    "I-OTHpart": 20,
    "I-PER": 21,
    "I-PERderiv": 22,
    "I-PERpart": 23,
    "O": 24
  },
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 512,
  "model_type": "bert",
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "pad_token_id": 0,
  "pooler_fc_size": 768,
  "pooler_num_attention_heads": 12,
  "pooler_num_fc_layers": 3,
  "pooler_size_per_head": 128,
  "pooler_type": "first_token_transform",
  "type_vocab_size": 2,
  "vocab_size": 119547
}

05/29/2020 11:21:51 - INFO - transformers.configuration_utils -   loading configuration file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-multilingual-cased-config.json from cache at /home/ll/.cache/torch/transformers/45629519f3117b89d89fd9c740073d8e4c1f0a70f9842476185100a8afe715d1.65df3cef028a0c91a7b059e4c404a975ebe6843c71267b67019c0e9cfa8a88f0
05/29/2020 11:21:51 - INFO - transformers.configuration_utils -   Model config BertConfig {
  "architectures": [
    "BertForMaskedLM"
  ],
  "attention_probs_dropout_prob": 0.1,
  "directionality": "bidi",
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 512,
  "model_type": "bert",
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "pad_token_id": 0,
  "pooler_fc_size": 768,
  "pooler_num_attention_heads": 12,
  "pooler_num_fc_layers": 3,
  "pooler_size_per_head": 128,
  "pooler_type": "first_token_transform",
  "type_vocab_size": 2,
  "vocab_size": 119547
}

05/29/2020 11:21:52 - INFO - transformers.tokenization_utils -   loading file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-multilingual-cased-vocab.txt from cache at /home/ll/.cache/torch/transformers/96435fa287fbf7e469185f1062386e05a075cadbf6838b74da22bf64b080bc32.99bcd55fc66f4f3360bc49ba472b940b8dcf223ea6a345deb969d607ca900729
05/29/2020 11:21:54 - INFO - transformers.modeling_tf_utils -   loading weights file https://cdn.huggingface.co/bert-base-multilingual-cased-tf_model.h5 from cache at /home/ll/.cache/torch/transformers/273ed844d60ef1d5a4ea8f7857e3c3869d05d7b22296f4ae9bc56026ed40eeb7.1b4841f14bf42137fc7ecee17a46c1b2f22b417f636347e4b810bd06dd9c45ea.h5
2020-05-29 11:21:55.520823: F ./tensorflow/core/kernels/random_op_gpu.h:232] Non-OK-status: GpuLaunchKernel(FillPhiloxRandomKernelLaunch<Distribution>, num_blocks, block_size, 0, d.stream(), gen, data, size, dist) status: Internal: invalid configuration argument
Aborted (core dumped)
```

## Expected behavior

I was trying to run the training of transformer ner extraction 

## Environment info

     
- `transformers` version: 
- Platform: Ubuntu 16.04.4 LTS
- Python version: Python 3.6.10 :: Anaconda, Inc.
- PyTorch version (GPU?): 
- Tensorflow version (GPU?): 2.2.0 GPU
- Using GPU in script?: yes
- Using distributed or parallel set-up in script?:

**Comment by jx669** [comment]
Delete. It may be a hardware issue. When I changed it to Colab, this problem disappeared
